scripts/Calc_hru_wgtavg_nc.py

```diff
@@ -86,11 +87,11 @@ def writeNetCDFData(fn, var, varname):
   dim1size=var.shape[0]
   dim2size=var.shape[1]
  
-  dim_1 = ncfile.createDimension('dim1',dim1size )  # hru axis
-  dim_2 = ncfile.createDimension('dim2',dim2size )  # hru axis
+  dim_1 = ncfile.createDimension(TDIM,dim1size )  # time axis
+  dim_2 = ncfile.createDimension(HDIM,dim2size )  # hru axis
 
   # Define a 2D variable to hold the var
-  val = ncfile.createVariable(varname,'f4',('dim1','dim2'))
+  val = ncfile.createVariable(varname,'f4',(TDIM,HDIM),zlib=True,complevel=1)
```

Comment:
Would there be a reason to make compression level a user supplied parameter? Are the state files easily readable if not compressed, but not if they are compressed? Will uncompressed state files work better more easily with DA? Just some questions about usability later.

Reply:
Sorry Andy, I don't know why this python script change was sneaked into pull request.  This python script has been changed and merged in different pull request.     so